### PR TITLE
Fixed typos in EvalAI-ngx

### PR DESCRIPTION
**Pull request by jayaike**
I fixed a few lines where there were typos or wrong capitalization.

@Ram81 @pushkalkatara @RishabhJain2018 @yashdusing @Sanji515 @vkartik97 Please take a look

### CODE REVIEW

**Review by krtkvrm — APPROVED** [review]

**Comment by pushkalkatara** [comment]
@nsjcorps Please update the branch with the base branch.

**Comment by jayaike** [comment]
@pushkalkatara Done!!

File: src/app/components/get-involved/get-involved.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h4 class="get-involved-title align-center fw-regular">Get Involved</h4>
   <div class="get-involved-description">
     <p class="fw-light content-section">
       Thanks for your interest in helping out with the EvalAI project! We're a
-      team of volunteers around the world who want to reduce the barrier to
+      team of volunteers around the world who want to reduce the barrier of
```
Comment (RishabhJain2018) [inline src/app/components/get-involved/get-involved.component.html]: Please revert this change.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: src/app/components/get-involved/get-involved.component.html (hunk 1 above)
Comment (jayaike) [reply src/app/components/get-involved/get-involved.component.html]: Alright. Will do now @RishabhJain2018

Comment (jayaike) [reply src/app/components/get-involved/get-involved.component.html]: I am also updating the branch

Comment (jayaike) [reply src/app/components/get-involved/get-involved.component.html]: I am also updating the branch

**Comment by jayaike** [comment]
Done!! @RishabhJain2018

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**